docs/docs/troubleshooting/SUPPORT_BOUNDARIES.md

```diff
@@ -61,3 +61,12 @@ Through the `values.yaml` the database is provided as a connection string with t
 !!!example ""
 
     See an example of [provisioning databases on cloud-based providers](../examples/database/CLOUD_PROVIDERS.md).
+
+## Bamboo - custom remote agents
+If creating and using custom remote docker agents, you're required to ensure that they are configured, built and working as expected. 
```

Comment:
What does the `as expected` mean?

Reply:
Well, in this context, `as expected`, means the resultant image behaves as you the owner expects it would.